task1/Person.java

```diff
@@ -0,0 +1,17 @@
+package task1;
+
+public class Person {
+    private String name;
+
```

Comment:
I would keep the age instance variable (and do this.age = age in the constructor) as it it passed in the exercise description